services/configuration/src/tests/mod.rs

```diff
@@ -199,16 +209,23 @@ fn test_apply_second_configuration() {
 fn test_apply_with_increased_majority() {
     let mut testkit = TestKitBuilder::validator()
         .with_validators(6)
-        .with_service(ConfigurationService {})
+        .with_service(ConfigurationService {
+            config: ConfigurationServiceConfig::default(),
+        })
         .create();
 
     // Applying the first configuration with custom majority count.
     let cfg_change_height = Height(5);
     let new_cfg = {
         let mut cfg = testkit.configuration_change_proposal();
         cfg.set_service_config("dummy", "First cfg");
-        cfg.set_majority_count(Some(6));
         cfg.set_actual_from(cfg_change_height);
+        cfg.set_service_config(
+            "configuration",
+            ConfigurationServiceConfig {
+                majority_count: Some(6),
```

Comment:
What do you think about rename `majority_count` to `quorum`?

Reply:
We may discuss this question and implement it in scope of the new PR.